### PR TITLE
Upstream parity 1.1.22

### PR DESCRIPTION
**Pull request by polygoblyn**
Please leave comments on anything that looks stinky

### CODE REVIEW

File: .github/workflows/stale_prs.yml [hunk 1]
```diff
@@ -14,5 +14,5 @@ jobs:
         stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being closed by a maintainer if it is not updated or reviews are not addressed. If your PR is closed as stale, feel free to open a new one after dealing with the issues. This may also be an indication that the maintainers do not have interest in this change, you can try to convince them otherwise, or persist in the doomed world you have created."
         stale-pr-label: 'Stale'
         exempt-pr-label: 'Needs Review'
-        days-before-stale: 30
+        days-before-stale: 100000
```
Comment (Zonespace27) [inline .github/workflows/stale_prs.yml:17]: Intended?

Comment (Kapu1178) [reply .github/workflows/stale_prs.yml:17]: We disabled auto stale.

Comment (Merlin1230) [reply .github/workflows/stale_prs.yml:17]: I never even enabled that workflow on here

**Comment by Merlin1230** [comment]
why didn't you run the checks aaaaaa

**Comment by nednaZ** [comment]
Antag Greentext Removal, 5955/1521ff2, is one we blocked last time.

**Comment by Merlin1230** [comment]
deltastation still throws an error, will fix soon

**Pull request merged**